habitat-baselines/habitat_baselines/rl/hrl/hl/neural_policy.py

```diff
@@ -191,6 +196,8 @@ def get_next_skill(
             if should_plan != 1.0:
                 continue
             use_ac = self._all_actions[skill_sel[batch_idx]]
+            if baselines_logger.level >= logging.DEBUG:
```

Comment:
Do you need this? I thought .debug already checks if the level is correct.

Reply:
Yes, this is needed because I want to avoid any cost of formatting the `use_ac` value.